pkg/rtc/subscriptionmanager.go

```diff
@@ -521,7 +539,7 @@ func (m *SubscriptionManager) subscribe(s *trackSubscription) error {
 }
 
 func (m *SubscriptionManager) unsubscribe(s *trackSubscription) error {
-	s.logger.Debugw("executing unsubscribe")
+	s.logger.Debugw("executing unsubscribe", "track", s.trackID)
```

Comment:
`trackID`? Also, this log line already seems to have `trackID`, maybe it is one of the track's logger has the trackID set?

Reply:
ah yeah, forgot about that.. will revert.